src/openrct2/ride/RideData.h

```diff
@@ -123,6 +123,8 @@ struct RideTypeDescriptor
     uint8_t MaxMass;
     /** rct2: 0x0097D7C8, 0x0097D7C9, 0x0097D7CA */
     rct_ride_lift_data LiftData;
+    // rct2: 0x0097CD1E
+    rating_tuple Ratings;
```

Comment:
Could you rename the field to RatingsMultipliers?

And could you rename the struct and its members to use TitleCase?

Reply:
Should I change ride_rating to RideRating?

Reply:
No, leave the `ride_rating` struct as-is for now. Otherwise the diff will get too big.